### PR TITLE
Monitoring Code for QC8

### PR DESCRIPTION
**Pull request by simonepv**
## Description

1- Monitoring code is independent from CMSSW and uses only python 2.7.5
2- Code cleaned. I left only parts useful for an eventual future debug
3- BitStatus.txt contains the meaning of status bits for HV A1515 and LV boards A3016 and A3016HP
4- StatusVmon904HV.py used to monitor HV system on QC8
5- StatusVmon904LV.py used to monitor LV system on QC8

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

Code has been tested comparing retrieved data with data stored on the INT2R database
The guide to use this code is on https://twiki.cern.ch/twiki/bin/view/CMS/QC8MonitoringHVLV

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by bdorney — APPROVED** [review]

**Comment by bdorney** [comment]
@jsturdy @mexanick review requested.

**Comment by bdorney** [comment]
Related too: https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/222

File: DCS_STATUS/StatusVmon904HV.py [hunk 1]
```diff
@@ -0,0 +1,1266 @@
...
+			statusNameList.append(status_name)
+			#print (listAllMainframeMappings[startIdx + contMaps][indexTB], " ", listAllBoardMappings[startIdx + contMaps][indexTB], " ", channel)
+		print "\nimonNameList", imonNameList
+		print "vomnNameList", vmonNameList
+		print "statusNameList", statusNameList
+	
+
+		#this is the connection to DB, and the contact point to DB should be input to the script 
+		#to avoid the have explicitely the pw in the script (the schema is the first field, then psw and after @ is the server ) 
+		db = cx_Oracle.connect('GEM_904_COND/904CondDB@INT2R')
```
Comment (jsturdy) [inline DCS_STATUS/StatusVmon904HV.py]: credentials should not be committed

File: DCS_STATUS/StatusVmon904HV.py [hunk 2]
```diff
@@ -0,0 +1,1266 @@
...
+			vmonNameList.append(vmon_name)
+
+		for contMaps in range( usedMaps ):
+			status_name="'cms_gem_dcs_1:CAEN/"+listAllMainframeMappings[startIdx + contMaps][indexTB]+"/"+listAllBoardMappings[startIdx + contMaps][indexTB]+"/"+channel+".actual.status'"
+			#this string has to be saved only one time
+			if statusNameList.count( status_name ) == 1:
+				continue
+			statusNameList.append(status_name)
+			#print (listAllMainframeMappings[startIdx + contMaps][indexTB], " ", listAllBoardMappings[startIdx + contMaps][indexTB], " ", channel)
+		print "\nimonNameList", imonNameList
```
Comment (jsturdy) [inline DCS_STATUS/StatusVmon904HV.py]: please uniformly use `print(str)`, you have mixed `print str` and `print(str)`

File: DCS_STATUS/StatusVmon904HV.py [hunk 3]
```diff
@@ -0,0 +1,1266 @@
...
+                   #print day
+                   hour = str(currentTsVmon)[11:13]
+                   #print hour
+                   minute = str(currentTsVmon)[14:16]
+                   #print minute
+                   second = str(currentTsVmon)[17:]
+                   #print second
+                   micro = str(currentTsVmon)[20:]
+                   #print micro
+
```
Comment (jsturdy) [inline DCS_STATUS/StatusVmon904HV.py:805]: consider using a python native library for this, e.g., [this](https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior) and [this](https://docs.python.org/2/library/datetime.html#datetime.datetime.strptime)

File: DCS_STATUS/StatusVmon904HV.py [hunk 4]
```diff
@@ -0,0 +1,1266 @@
...
+                Vmontg1.SetMarkerColor(4)
+                Vmontg1.SetMarkerStyle(21)
+		Vmontg1.SetMarkerSize(1)
+                Vmontg1.SetName("HV_VmonChamber"+chamberList[indexB]+"_"+channelList[indexC]+"_UTC_time")
+                Vmontg1.SetTitle("HV_VmonChamber"+chamberList[indexB]+"_"+channelList[indexC]+"_UTC_time")
+                #Vmontg1.GetXaxis().SetTitle("time [s]")
+                Vmontg1.GetYaxis().SetTitle("Vmon "+chamberList[indexB]+" "+channelList[indexC]+" [V]")
+                #Vmontg1.Draw("ACP")
+		Vmontg1.GetXaxis().SetTimeDisplay(1)
+                Vmontg1.GetXaxis().SetTimeFormat("#splitline{%y-%m-%d}{%H:%M:%S}%F1970-01-01 00:00:00")
```
Comment (jsturdy) [inline DCS_STATUS/StatusVmon904HV.py]: clean up the whitespace here and elsewhere (convert all `tab` characters to an appropriate number of spaces), as it appears when viewing here that the alignment is wrong

File: DCS_STATUS/BitStatus.txt [hunk 5]
```diff
@@ -0,0 +1,31 @@
+HV boards (CAEN A1515)
```
Comment (jsturdy) [inline DCS_STATUS/BitStatus.txt]: is this file imported or used anywhere?
if not, its information would better be served in a user guide

**Review by jsturdy — CHANGES_REQUESTED** [review]
* remove secrets (from history, not just from file)
* cleanup whitespace
* utilize `python3` friendly `print()`

**Comment by simonepv** [comment]
Hi Jared,

can you tell me how to remove secrets from history?
Because I have no idea on how to do it

Kind regards

Simone

Il giorno ven 21 giu 2019 alle ore 12:19 Jared Sturdy <
notifications@github.com> ha scritto:

**Comment by jsturdy** [comment]
@simonepv you can try [this](https://help.github.com/en/articles/removing-sensitive-data-from-a-repository)

**Review by jsturdy — APPROVED** [review]

**Pull request merged**